internal/dumper/dumper_test.go

```diff
@@ -14,6 +14,17 @@ import (
 	qt "github.com/frankban/quicktest"
 )
 
+func createFieldRow(name, extra string) []sqltypes.Value {
```

Comment:
Usually in go we prefix variables in a test file with `test`. So this could be `testRow()`. It's read nicely and is also short.